lib/tasks/data/truncate_data.rake

```diff
@@ -9,6 +9,9 @@ namespace :ofn do
 
       sql_delete_from "
         spree_inventory_units #{where_order_id_in_orders_to_delete}"
+      sql_delete_from "
+        spree_inventory_units
+        where shipment_id in (select id from spree_shipments #{where_order_id_in_orders_to_delete}"
```

Comment:
This seems to be missing a closing parenthesis.

Reply:
I added it now @kristinalim.

Reply:
ah, I dont know how did this happen. I used this code before but must have made the mistake when committing it...
thanks Maikel.
I tested it now locally and it is working. With the extra parenthesis :-)